vm/systemSmartContracts/delegation.go

```diff
@@ -1317,7 +1340,7 @@ func getSuccessAndUnSuccessKeys(returnData [][]byte, blsKeys [][]byte) ([][]byte
 func verifyIfBLSPubKeysExist(listKeys []*NodesData, arguments [][]byte) error {
 	duplicates := checkForDuplicates(arguments)
 	if duplicates {
-		return vm.ErrBLSPublicKeyMismatch
+		return vm.ErrDuplicatesFoundInArguments
```

Comment:
vm.ErrDuplicatesFoundInArguments also in line 1349

Reply:
changed